multiqc/modules/biscuit/biscuit.py

```diff
@@ -0,0 +1,688 @@
+                if m is not None:
+                    data[m.group(1)] = int(m.group(2))
+
+        if fn.endswith('_isize_score_table.txt'):
+            data['I'] = {}      # insert size
+            data['S'] = {}      # SW-score
+
+            for l in f.splitlines():
+                fields = l.split()
+                if fields[0] == 'I':
+                    data[fields[0]][int(fields[1])] = float(fields[2]) * 100
+                elif fields[0] == 'S':
+                    data[fields[0]][fields[1]] = float(fields[2])
+                
+        return data
+    
+    def chart_biscuit_mapq(self):
+
+        # fraction of optimally mapped reads
+        pd = {}
+        for sid, dd in self.mdata['mapq'].items():
+            if not sid.endswith('_mapq_table.txt'):
+                continue
+            sid = sid.replace('_mapq_table.txt','');
+            pd[sid] = {'OAligned':0, 'SAligned':0, 'UAligned':1}
+            for mapq, cnt in dd.items():
+                if mapq == 'unmapped':
+                    pd[sid]['UAligned'] += int(cnt)
+                elif int(mapq) >= 40:
+                    pd[sid]['OAligned'] += int(cnt)
+                else:
+                    pd[sid]['SAligned'] += int(cnt)
+
+        self.add_section(
+            name = 'Mapping Summary',
+            anchor = 'biscuit-mapping',
+            plot = bargraph.plot(pd, OrderedDict([
+                ('OAligned', {'name':'Optimally Aligned Reads', 'color': '#8bbc21'}),
+                ('SAligned', {'name':'Suboptimally Aligned Reads', 'color': '#f7a35c'}),
+                ('UAligned', {'name':'Unaligned Reads', 'color': '#000000'})
+            ]), {'id':'biscuit_mapping_summary',
+                 'title':'BISCUIT: Mapping Summary',
+                 'ylab':'Number of Reads',
+                 'cpswitch_c_active': True,
+                 'cpswitch_counts_label': '# Reads'
+            })
+        )
+
+        # mapping strand distribution
+        pd = dict([(k.replace('_strand_table.txt', ''),v) for k,v in self.mdata['mapq'].items() if k.endswith('_strand_table.txt')])
+        self.add_section(
+            name='Mapping Strand Distribution',
+            anchor='biscuit-strands',
+            description = "<p>This plot shows the distribution of strand of mapping and strand of bisulfite conversion.</p>",
+            plot = bargraph.plot(pd, OrderedDict([
+                ('++', {'name':'Waston-Aligned, Waston-Bisulfite Conversion', 'color': '#F53855'}),
+                ('+-', {'name':'Waston-Aligned, Crick-Bisulfite Conversion', 'color': '#E37B40'}),
+                ('-+', {'name':'Crick-Aligned, Waston-Bisulfite Conversion', 'color': '#46B29D'}),
+                ('--', {'name':'Crick-Aligned, Crick-Bisulfite Conversion', 'color': '#324D5C'})
+            ]), {'id':'biscuit_strands',
+                 'title':'BISCUIT: Mapping Strand Distribution',
+                 'ylab':'Number of Reads',
+                 'cpswitch_c_active': True,
```

Comment:
`True` is the default - maybe you mean `False` so that the bars display percentages by default? This would make sense to me, as we're looking for differences in proportions here, rather than counts.

Reply:
Yes, I changed it to False. Thanks